config/default.yaml

```diff
@@ -120,6 +120,9 @@ user:
 transcoding:
   enabled: true
   threads: 1
+  concurrency: 1
+  niceness: 15
+  ttl: 48 # 2 days, transcoding could be long
```

Comment:
Use a string instead so users can specify '1 hour' or '2 days' (take example on `check_interval`)

Reply:
ok sounds good

Reply:
once it parses it on save it becomes an int and I can't seem to make it stay the duration string on save unless theres another function i don't know about yet which turns a int into a duration